fms/add_sensor.py

```diff
@@ -0,0 +1,227 @@
+import os
+import subprocess
+
+from happi import Client
+from happi.errors import EnforceError, SearchError
+
+from .happi.containers import FMSRaritanItem, FMSSRCItem
+from .utils import TypeEnforcer as te
+
+fms_happi_database = os.environ["HAPPI_CFG"]
+
+with open(fms_happi_database, "r") as file:
+    lines = file.readlines()
+    for line in lines:
+        if "path" in line:
+            fms_happi_database = line.split("=")[1].strip()
+
+sensor_name = 0
```

Comment:
Nitpick: this variable is confusing because it's a name but it's an int and `sensor_name` is re-used as an input to add_sensor

Reply:
This is confusing. Good catch. Will fix before merge.

Reply:
updated.